docs/UserGuide.md

```diff
@@ -183,4 +260,26 @@ There is no need to save manually.
 * **Select** : `select INDEX` <br>
   e.g.`select 2`
 
+* **Update** : '`update INDEX [NAME] [d/ENDDATE] [sd/STARTDATE] [r/PERIOD(DAY/WEEK/MONTH)] [t/TAG]...'
+  e.g. 'update 1 d/11/12/2011'
+
+* **Exit** : 'exit'
+
+* **Undo** : 'undo [NUMBER]'
+  e.g. 'undo 5'
```

Comment:
you mean the [] or having an example for undo 5?

the [] is listed at the beginning.
undo has an example for single use and multiple use versions.